vk-diff/src/groth16_vk.rs

```diff
@@ -0,0 +1,142 @@
+use anyhow::{anyhow, Result};
```

Comment:
add a TODO so we don't forget to merge it with where it's copied from?

Reply:
Sure! 🙏